config/default-spinnaker-local.yml

```diff
@@ -45,6 +45,22 @@ providers:
     #   name: my-cf-username
     #   password: my-cf-password
 
+  kubernetes:
+    # If you want to deploy some services to Kubernetes
```

Comment:
Missing comma at the end of this line.
Any reason not to add `(k8s)` for consistency with the other provider blocks?